src/main/java/com/itschool/eventmanagment/models/entities/Event.java

```diff
@@ -25,12 +26,10 @@ public class Event {
     private Long id;
     @Column
     private String name;
-    @Column (name = "event_date_time")
-    private LocalDateTime eventDateTime;
     @JsonDeserialize(using = LocalDateTimeDeserializer.class)
     @JsonSerialize(using = LocalDateTimeSerializer.class)
```

Comment:
Please consider to remove `@Json...` annotation from the entity. This class is used to represent the database structure, so is not directly interacting with JSON serialization/deserialization. We should keep those annotations only on DTOs.

Reply:
If I removing I will have a 500 error.

<img width="747" alt="Screenshot 2024-09-19 at 12 07 39" src="https://github.com/user-attachments/assets/89bc7ed6-b52c-4fac-8b99-a5769d87f871">
<img width="1503" alt="Screenshot 2024-09-19 at 12 07 47" src="https://github.com/user-attachments/assets/19a1d515-fe11-4235-bc67-14032841284f">